aedes.js

```diff
@@ -69,6 +69,7 @@ function Aedes (opts) {
   this.authorizePublish = opts.authorizePublish
   this.authorizeSubscribe = opts.authorizeSubscribe
   this.authorizeForward = opts.authorizeForward
+  this.preUnsubscribe = opts.preUnsubscribe
```

Comment:
You should add a `defaultPreSubscribe` function and add it to `defaultOptions`

Reply:
Ah, yes, good point. Thanks, done.

Reply:
Should that allow some async stuff like the others do? maybe adding a callbackk to it?